tags-config-service-impl/src/test/java/org/hypertrace/tag/config/service/TagConfigServiceImplTest.java

```diff
@@ -67,6 +86,19 @@ void test_createTag() {
     assertEquals(createTagsList, createdTagsList);
   }
 
+  @Test
+  void test_system_createTag() {
+    for (String key : systemTags) {
+      CreateTagRequest createTagRequest =
+          CreateTagRequest.newBuilder().setTag(CreateTag.newBuilder().setKey(key).build()).build();
+      try {
+        tagConfigStub.createTag(createTagRequest);
+      } catch (Exception e) {
+        assertEquals(Status.ALREADY_EXISTS, Status.fromThrowable(e));
```

Comment:
Please use below mechanism to validate exception -
`
@Test
void exceptionTesting() {
    Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
        throw new IllegalArgumentException("a message");
    });
    assertEquals("a message", exception.getMessage());
}
`